etc/template_config.xml

```diff
@@ -85,46 +85,52 @@ xsi:noNamespaceSchemaLocation="template_list.xsd">
             <name>New Shipment</name>
             <custom_template_source>sales_email/shipment/template</custom_template_source>
             <sort_order>14</sort_order>
+        </template>
+         <template>
+            <id>sales_email_shipment_guest_template</id>
+            <name>New Shipment</name>
```

Comment:
This name needs to be new New Guest Shipment. Right now, it looks the same as the regular shipment template on the page.